pkg/ccip/ocr/contract_transmitter.go

```diff
@@ -129,14 +120,14 @@ func (c *ccipTransmitter) Transmit(
 
 	c.lggr.Infow("Transmitting OCR report",
 		"txID", txID,
-		"from", w.Address().String(),
+		"from", w.WalletAddress().String(),
 		"to", c.offrampAddress,
 		"configDigest", hex.EncodeToString(configDigest[:]),
 		"seqNr", seqNr,
 		"reportBytes", len(reportWithInfo.Report),
 		"signatures", len(sigs),
 		"gasLimit", gasLimit,
-		"finalAmountTON", finalAmount,
+		"finalAmountTON", finalAmount.String(),
```

Comment:
nit: shall we do gasLimit.String() above as well ? Think it doesn't matter but to be consistent